client-sdk/engine/Evaluator.cs

```diff
@@ -114,7 +115,7 @@ private bool CheckDependencies(FeatureFlag featureFlag, UnlaunchUser user)
             return true;
         }
 
-        private long GetHash(string key)
+        private static long GetHash(string key)
```

Comment:
just curious, why use static? We're operating on objects/instances not on classes,

Reply:
Visual studio suggests me to use static, and it is a private method, so I just made it so.